gameflow.md

```diff
@@ -0,0 +1,64 @@
+# Gameflow
+
+## Game Logic
+
+The selected level starts and presents the user a random syllable (Hiragna / Katakana) as well as four possible answers.
+The available syllables depend on the selected level.
+
+If the correct answer was selected, the game immediately goes to the next syllable.
+
+If a wrong answer was selected, the game shows a quick indicator that something was wrong, removes a heart container and then goes to the next syllable.
+
+If the required amount of syllables have been answered, the game navigates to the success screen and informs the player how many Gems / Jewels he won as well as if he unlocked new things like syllables.
+
+If the user runs out of Heart Containers the game navigates to the failure screen.
+
+## Game Segments
+
+**Syllable**
+
+A single piece of gameplay in which the player has to select the correct counterpart for a given syllable in latin character.
+
+**Level**
+
+A collection of syllables the user has to complete. At the end of the level the user will be awarded in the form of gems.
+
+**Gems / Jewels**
+
+Will be rewarded on completion of a level, no matter if the user succeeded or failed.
+Can be used to buy upgrades in the Gems / Jewels store. In this way the player can ease the growing complexitiy / difficulity of newly unlocked levels.
+
+**Heart Container**
+
+Represents the amount of wrong answers a user is allowed to give per level. Will be refilled on start of a new level. Initially, 3 Heart Containers are given.
+
+**Upgrades**
+
+- Additional Heart Containers
+- New and more valubale Gems / Jewels
```

Comment:
```suggestion
- New and more valuable gems / jewels
```

Reply:
Do we perhaps want to add information here what this means? Like, what is the value of a Gem?
Also, shouldn't we settle for one name?

Reply:
Regarding value: We could, but this should be discussed first I think.
Regarding name: We should, but I figured this is worth discussing.